src/main/java/com/offdk/play/model/SlackCommand.java

```diff
@@ -5,9 +5,12 @@
   private String teamId;
   private String teamDomain;
   private String channelId;
+  private String channelName;
   private User commandUser;
+  private String command;
   private String text;
   private String responseUrl;
+
```

Comment:
Add a constructor so you could potentially remove many of the setters. If nothing changes after the initial values are set, make the variables final.